core-bundle/src/Resources/config/services.yml

```diff
@@ -817,6 +817,18 @@ services:
         tags:
             - { name: kernel.reset, method: reset }
 
+    contao.twig.image_extension:
```

Comment:
We should use the FQCNs instead of a service ID if possible.

Reply:
You wouldn't want to autowire this, would you?

Reply:
I don't know what you mean. 😄 We want to use FQCNs for all services in the future as recommended by Symfony.

Reply:
Changed to FQCN in ff2538a40e248d02da8df72f424097c168beaf86